signatures/golang/kubernetes_api_connection.go

```diff
@@ -43,6 +43,14 @@ func (sig *K8sApiConnection) GetSelectedEvents() ([]detect.SignatureEventSelecto
 	}, nil
 }
 
+func (sig *K8sApiConnection) GetFilters() ([]protocol.Filter, error) {
+	return []protocol.Filter{
+		{Field: "execve.container", Operator: protocol.Equal, Value: []interface{}{true}},
+		{Field: "security_socket_connect.container", Operator: protocol.Equal, Value: []interface{}{true}},
+		{Field: "event", Operator: protocol.Equal, Value: []interface{}{"execve", "security_socket_connect"}},
```

Comment:
Would we need to add filter to state the source type and events? seems really redundant due to the fact that we already state it in the event selector...
Couldn't you get that from the event selectors automatically?

Reply:
Also can we add a helper for each operator? that way we can use:
`helpers.equals("event", ["execve", "security_socket_connect"])` and it will return the appropriate filter object.
This will be way more readable and will help us convert our signatures.

Reply:
I'm thinking maybe separating the Filter from protocol and adding a filters package to types. Then we could define these helpers there WDYT?

Reply:
Sounds good

Reply:
Done.